work-scheduler-web/src/main/webapp/WEB-INF/jsp/registration.jsp

```diff
@@ -0,0 +1,33 @@
+<%--
+  Created by IntelliJ IDEA.
+  User: Zeller
+  Date: 2017. 08. 31.
+  Time: 19:48
+  To change this template use File | Settings | File Templates.
+--%>
+<%@ page contentType="text/html;charset=UTF-8" language="java" %>
+<html>
+    <head>
+        <title>Title</title>
+    </head>
+    <body>
+        <form method="post" action="${pageContext.request.contextPath}/registration">
+            <label for="personal-name">Employee name:</label>
+            <input type="text" name="name" id="personal-name"/>
+
+            <label for="personal-date">Personal date:</label>
```

Comment:
Personal date doesn't really make any sense.
This form field should hold the birth date data. Rename the `for` attribute of the `label`, the label text itself and the corresponding `id` of the `input` to something like "Birth date" and "birth-date".